### ISSUE TITLE
License

### ISSUE THREAD

**Issue by dbenjaminmiller** [rstub/fpl#4]
https://github.com/URWTypeFoundry/Core_35

See here: the basis for these fonts has been relicensed, so we may now change as well.

**Comment by rstub** [comment]
Here a link to the commit: https://github.com/URWTypeFoundry/Core_35/commit/8a814a78792cea85d8d1f7c5806e08d22fa4b5a3

**Comment by rstub** [comment]
The above repo is defunct, but there is a fork at https://github.com/twardoch/urw-core35-fonts.

**Comment by dbenjaminmiller** [comment]
Yep. By the way, [Nimbus Roman No. 9 SC was also added by URW in 2019](https://github.com/twardoch/urw-core35-fonts/pull/1), but that's not really relevant for fpl, since we are only using Palladio/P052. Though as mentioned in the repo, the tri-lience (LPPL/OFL/AGPL+exception) only directly applies to version 2.0 of the URW fonts, so, I suppose, you could only change the license by re-basing the font on the URW 2.0 release. (The URW 2.0 release is virtually identical to the 1.0 release, besides the addition of more characters, though.)

In any case, I ended up making my own font later in 2019, dbenjaminmiller/domitian, which (respectfully) I would say replaces fpl, using the URW 2.0 font as a basis. I drew new small caps and old-style figures myself. I also added the font to CTAN, and it's [in the LaTeX Font Catalogue](https://tug.org/FontCatalogue/domitian/). While I didn't copy any outlines from any existing digital font, my OSF/SC characters are meant to match the design used by the Adobe/Linotype/etc. releases of Palatino, rather than being distinctive like fpl's.

I later drew [GFS Heraklit](https://www.greekfontsociety-gfs.gr/typefaces/20th_21st_century#GFS_Heraklit) for the Greek Font Society. GFS Heraklit only has a regular upright font (maybe one day I'll design a family, but probably not). It reuses the OSF/SC I drew for Domitian, but I then added a full set of Greek characters which were the first digital version of Zapf's Heraklit font (which was the Greek counterpart of Palatino).

**Comment by rstub** [comment]
Indeed, there is nothing to change since version 1 of the URW Base 35 fonts, which was the basis of these fonts, is covered by LPPL/GPL+exception, which is already used here. So there is nothing to be done and I should probably either close this issue or migrate it to https://github.com/rstub/fplneu. If I ever come back to FPL Neu, then I will start fresh and use the tripple licensed version 2 of the fonts.

I am not sure I understand in what sense you see Domitian as a replacement for FPL. Some history: about 20 years ago the late Walter Schmidt (PSNFSS maintainer at that time) added support for the Adobe Palatino SC/OsF to mathpazo. At that time I thought it would be nice to have a free alternative for these fonts, which I wanted to use for typesetting my PhD thesis. So I created the FPL fonts with the explicit goal to use the same metrics as Adobe Palatino SC/OsF, which were then included in PSNFSS. This is why there is no separate entry for FPL in the LaTeX Font Catalogue and is instead used on the page for [URW Palladio](https://tug.org/FontCatalogue/urwpalladio/). BTW, I strongly disagree with the notion that TeX Gyre Pagella is an "enhanced version", but that battle was fought and lost a long time ago. although it is one of the main reasons why I have basically stopped working on fonts ...

As for you Domitian: From all I can tell it is a very well made font. Especially the bold-italic OsF are better than mine in FPL. However, if you want to replace FPL, than you have to convince the current PSNFSS maintainer to switch from using FPL (and URW Palladio?) to Domitian in the relevant map files. That might make sense if they are metrics compatible.

Of course, their are changes one wants to do that are not metrics preserving, including some feedback Walter and I got from Hermann Zapf himself, which we wanted to put into FPL Neu. But see above ...
